### ISSUE TITLE
CMMC

### ISSUE THREAD

**Issue by trevorbryant** [redteam-project/sckg#19]
I've extracted the controls in CMMC v1.02 for usability. We'll figure out what needs to be correlated from CMMC <> 800-171 <> 800-53.

https://github.com/trevorbryant/cmmc-controls

**Comment by jason-callaway** [comment]
Thanks, Trevor!

How are these mapped to 800-171 or 800-53 controls? I assume the "practice number" column is the unique identifier for each CMMC control?

**Comment by jason-callaway** [comment]
Also, could you please PR this into this repo's [data/regimes](https://github.com/redteam-project/sckg/tree/master/data/regimes) directory?

**Comment by trevorbryant** [comment]
> How are these mapped to 800-171 or 800-53 controls? I assume the "practice number" column is the unique identifier for each CMMC control?

Yes, the `practice number` is the unique control identifier for CMMC. 

I haven't yet taken the time to map the CMMC controls to 800-53. I don't anticipate it being a level of effort considering CMMC is focused on 800-171, which derives from 800-53.

We'll see.
